routing/routing_session.cpp

```diff
@@ -417,11 +417,13 @@ void RoutingSession::GetRouteFollowingInfo(FollowingInfo & info) const
 
   info.m_exitNum = turn.m_exitNum;
   info.m_time = static_cast<int>(std::max(kMinimumETASec, m_route->GetCurrentTimeToEndSec()));
-  RouteSegment::RoadNameInfo sourceRoadNameInfo, targetRoadNameInfo;
+  RouteSegment::RoadNameInfo sourceRoadNameInfo, targetRoadNameInfo, nextRoadNameInfo;
   m_route->GetCurrentStreetName(sourceRoadNameInfo);
   GetFullRoadName(sourceRoadNameInfo, info.m_sourceName);
   m_route->GetNextTurnStreetName(targetRoadNameInfo);
   GetFullRoadName(targetRoadNameInfo, info.m_targetName);
+  m_route->GetNextNextTurnStreetName(nextRoadNameInfo);
+  GetFullRoadName(nextRoadNameInfo, info.m_nextStreetName);
```

Comment:
As I mentioned above, here we assign m_nextStreetName the same way as m_targetName.

But in Android code we use m_displayedStreetName.